docs/input/docs/index.cshtml

```diff
@@ -1,11 +1,10 @@
 ---
 Title: Documentation
+Order: 10
 ---
-<p>This user guide, like Cake.Recipe itself, is under very active development. Some parts of it aren't 
-documented as completely as they need to be, but we gladly accept your contributions.</p>
-
-<p>We need your help to improve the documentation for Cake.Recipe, so if there is something that you 
-would like to add then you can edit the content directly on GitHub.</p>
+<p>
+    We gladly accept your contributions. If there is something that you would like to add then you can edit the content directly on GitHub.
```

Comment:
Only partially true, they can edit on GitHub but they would need to open a PR to have any updates being available on the documentation site.
Maybe linking to `how to contribute` documentation as well?